### PR TITLE
Added a response for why_dont_vegans_eat_honey.md

### PR DESCRIPTION
**Pull request by Fishy-x**
Also fixed line 3 in why_dont_vegans_eat_honey.yaml. Unless that was supposed to be there. 
Let me know if there are any grammatical errors or if there is something you would like for me to delete/add.

### CODE REVIEW

File: knowledge/responses/why_dont_vegans_eat_honey.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
-We don't currently have a response for this myth! Please let us know if you'd like to contribute to writing the response to this myth by posting here: 
+Bee's make honey for bee's, and not for humans.
```
Comment (suanchinyeo) [inline knowledge/responses/why_dont_vegans_eat_honey.md:1]: I would structure this in point form so it's more readable, for example:

**Bees make honey for bees, not for humans**
[ paragraph about why bees make honey]

**Honey farming is bad for the environment**
[paragraph about honey bees and bumblebees]

**Honey farming practices are cruel**
[if you want, you can add in some stuff about bees being sentient and intelligent and live in a social environment]
[paragraph about how we take honey from them, the substitute we give them is not good enough for their health]
[paragraph about killing bees/hives]
[add a paragraph about how male bees are crushed to get their semen and how queen bees have their wings clipped, you can find this information on earthling ed's why vegans don't eat honey video]

**Alternatives to honey**
[paragraph about plant based syrups and sweeteners]

Other than that I would go through the text and change bee's to bees when it's plural and bee's when it's possessive.

Love the links and the writing, very concise and easy to read!

**Comment by suanchinyeo** [comment]
Hi @Fishy-x, thank you so much for writing this for us. I love the simple and to the point writing. I have left some notes on minor formatting changes and a request to add in information about queen bees and male bees, other than that it's great.

Thank you so much again :)

**Comment by Fishy-x** [comment]
I double checked my grammar this time and I believe the errors have been fixed. I tried to follow the format you provided and added what you suggested. Let me know if you think the response is too long or too short. Thanks!

**Comment by cvqluu** [comment]
Great job with this, thanks. 

I just made a few small editing changes, but the response you wrote is largely unchanged, just rearranged.

Ready to be merged

**Comment by cvqluu** [comment]
https://www.reddit.com/r/testanimalsupportbot/comments/mal9jl/newtestpost/gysyaj4/?context=10000

**Pull request merged**